index.html

```diff
@@ -48,13 +63,149 @@ <h3>LET'S CONNECT</h3>
+                <p>Fullstack</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
+             <p class="writeup-div">Experimental content creation feature that allows users to add to an 
+              existing story ovwr the cause of a day without spamming their friends
+             </p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+            <div class="mobile-image">
+              <img src="desktop3.svg" alt="" class="card-img-img">
+            </div>
+           </div>
+
+           <div class="mobile-version">
+            <div class="mobile-image">
+              <img src="desktopimg.svg" alt="" class="card-img-img">
+            </div>
+            <div class="mobile-section-div">
+              <h1>Facebook 360</h1>
+              <div class="canopy-div">
+                <p>facebook</p>
+                <img src="Counter.svg" alt="" />
+                <p>fullstack</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
+             <p class="writeup-div">Experimental content creation feature that allows users to add to an 
+              existing story ovwr the cause of a day without spamming their friends</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+           </div>
+
+           <div class="mobile-version">
+            <div class="mobile-section-div-left">
+              <h1>Uber Navigation</h1>
+              <div class="canopy-div">
+                <p>Uber</p>
+                <img src="Counter.svg" alt="" />
+                <p>Lead developer</p>
+                <img src="Counter.svg" alt="" />
+                <p>2018</p>
+              </div>
+             <p class="writeup-div">A daily selection of privately personalized reads; no accounts or sign ups required</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+            <div class="mobile-image-right">
+              <img src="desktop2.svg" alt="" class="card-img-img">
+            </div>
+           </div>
+        </section>
+
+        <section class="desktop-second-about">
+          <div class="desktop-about-left">
+            <h1 class="abouth1">About Myself</h1>
+            <p>Hello im a software developer, i can help you build a product, feature or website look through some of my work
```

Comment:
- [ ] I kindly suggest that the left side of your about me should take 40% of the width, and the right one the remaining 60% of the width. See image below


<img width="908" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/ba20225a-878d-4c77-bdb1-f05ed8f5e7dd">